repos/cli.proj

```diff
@@ -41,7 +41,13 @@
     <RepoApiImplemented>false</RepoApiImplemented>
     <SourceOverrideRepoApiImplemented>true</SourceOverrideRepoApiImplemented>
 
+    <!-- Temporarily disable auto-update: patch consistently conflicts with dependency changes. -->
+    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
+
+    <ProdConBlobFeedUrl>$([System.IO.File]::ReadAllText('$(ProdConFeedPath)').Trim())</ProdConBlobFeedUrl>
+
     <EnvironmentExternalRestoreSources>$(SourceBuiltPackagesPath)</EnvironmentExternalRestoreSources>
+    <EnvironmentExternalRestoreSources>$(EnvironmentExternalRestoreSources)%3B$(ProdConBlobFeedUrl)</EnvironmentExternalRestoreSources>
```

Comment:
I skipped ahead to a CLI-only build to see if this works: it doesn't. MSBuild changes the `https://` to `https:/`. I'm trying the usual escaping with `%2F` and `%252F` which don't seem to change the result, I think because the string leaves the MSBuild context and comes back into CLI as a property through the environment variable.

Reply:
We'll need a fix/patch in CLI to do this properly, I'm pretty sure. `%25252F` gets nice escaped `/`s in there, but CLI doesn't have an Unescape call to convert them into `/`s. There's also as far as I can tell no way to manually hook into the build to modify the nuget.config after CLI generates it.

In the meantime, I think I can do a "soft patch"/updater for CLI's `NugetConfigFile.targets` to add a line to the source code. I'm going to try that.